azure/functions/_http_wsgi.py

```diff
@@ -10,7 +10,7 @@
 
 
 class WsgiRequest:
-    _environ_cache: Dict[str, Any] = None
+    _environ_cache: Optional[Dict[str, Any]] = None
```

Comment:
For these `Optional` - Is the goal to make them strongly typed later in the future?

Reply:
Nice, good catch 👍

Reply:
Yes, this _environ_cache is a cache of WSGI Dict in memory. When cache miss, it will be assigned later.